src/build/brick.rs

```diff
@@ -318,13 +318,20 @@ impl Baked {
                 })
                 .collect()
         });
-        baked_bricks
+        let search_base = search_alias.with_base();
+        let baked = &baked_bricks
             .iter()
-            .filter(|(baked_alias, _)| alias.matches(baked_alias))
+            .filter(|(baked_alias, _)| search_base.matches(baked_alias))
             .min_by_key(|(brick_alias, _)| brick_alias.0.len())
-            .expect(&format!("no such brick: '{alias}'"))
-            .1
-            .clone()
+            .expect(&format!("no such brick: '{search_base}'")).1;
+        let mut thawed = baked.clone();
+        for face in &mut thawed.faces {
+            face.aliases.retain(|candidate| search_alias.matches(candidate));
+            if face.aliases.len() != 1 {
+                panic!("exactly one face should be retained");
```

Comment:
I'm starting to feel we throw panic!s and unwraps() around too cavalierly. We should do a panic-to-Result refactor in the future

Reply:
I think this one is just an assertion, but yes.